backend/src/processing/mod.rs

```diff
@@ -177,7 +177,10 @@ impl EventProcessor {
     /// Handle different event types
     async fn handle_event_types(&self, processed: &mut ProcessedEvent) -> Result<()> {
         let event_name = processed.event.raw.event_name.clone();
-        if processed.event.raw.is_custom_event {
+        if event_name.eq_ignore_ascii_case("cwv") {
+            processed.event_type = "cwv".to_string();
+            processed.custom_event_json = processed.event.raw.properties.clone();
```

Comment:
Suggestion: I would definitely prefer if we typed out the properties, as this `properties` is meant for `custom_event_json`, which in turn is meant to store arbitrary data for the custom events for users.
I think it'd be better if we added the `cwv` fields to the ClickHouse events table instead, as it'll give us much better validation / safety / performance 😄